src/DryIoc/Container.cs

```diff
@@ -46,6 +46,9 @@ THE SOFTWARE.
 #if !PCL && !NET35 && !NET40 && !NET403
 #define SUPPORTS_DELEGATE_METHOD
 #endif
+#if !NET35 && !PCL
```

Comment:
why not to drop net35 and pcl?

Reply:
Dropping them in V5.
There is v5-dev branch where Ibdid tgat allready. So the current fallback to lock will be removed in v5.